07-wt-multilevel-models-2.Rmd

```diff
@@ -166,7 +166,7 @@ makes it easy to do this. This function allows us to re-order the levels within
 ``` {r}
 dat <-
   dat %>%
-  mutate(course_id = fct_relevel(course_id, "course_idPhysA-S116-01"))
+  mutate(course_id = fct_relevel(course_id, "PhysA-S116-01"))
```

Comment:
did the former not work (with `course_id` prepended)? wondering if that was an artifact of an older dataset.

Reply:
confirming you were right here - thanks